js/apps/events-page/controllers/eventsPageCtrl.js

```diff
@@ -153,6 +155,12 @@
 				key: 'term',
 				val: vm.keywords
 			}]); // This will trigger a location change, therefore getting the new results
+
+			trackEvent({
+				action: 'Keyword Search',
+				category: 'Events',
```

Comment:
Do we want the category to be this generic? Maybe have a 'Search' category and have the label be the element name? That way we will know exactly where the event comes from.

Reply:
@jsmith176 category of "Events" is for BCPL Events which can be found here, https://www.bcpl.info/events-and-programs/list.html#!/

@danfox01 do you want me to clarify that this is for "BCPL Events" as the category?

Reply:
@martypowell I think that makes sense.

Reply:
@jsmith176 this has been updated and moved to the constants file.